### PR TITLE
T6 Get /api/users

### PR DESCRIPTION
**Pull request by RaymondBibby**
add completed end point:
 /api/users with passing test suite.

Happy path:
Status: 200 correctly serves up the users in the expected format with the expected properties.

Sad path:
No sad path --> Catchall implemented and tested on T4

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -167,4 +167,26 @@ describe("PATCH: /api/articles/:article_id", () => {
...
+
+describe("GET: /api/users", () => {
+    it("Status 200: responds with an array of user objects, each of which have the following properties: 'username', 'name' and 'avatar_url'. ", () => {
+        return request(app)
+            .get('/api/users')
+            .expect(200)
+            .then(( {body} ) => {
+                const { users } = body
+                expect(users).toBeInstanceOf(Array);
+
```
Comment (alexdevdavis) [inline __tests__/app.test.js:182]: consider testing for expected array length

File: controller/news_controller.js [hunk 2]
```diff
@@ -30,3 +31,27 @@ exports.patchArticleById = (req, res, next) => {
 }
 
 
+
+
```
Comment (alexdevdavis) [inline controller/news_controller.js:35]: whitespace suggests the need to extract controllers (and models, for that matter) out into their own files: topics, users, articles etc.

**Review by alexdevdavis — APPROVED** [review]
does your sad path test against this endpoint? include tests so that you know (the fact that you won't have to necessarily implement any more logic to pass them is a bonus, but you ought attempt the RGR cycle nonetheless)

File: __tests__/app.test.js (hunk 1 above)
Comment (RaymondBibby) [reply __tests__/app.test.js:182]: Test criteria added:
expect(users.length).toBe(4)

**Pull request merged**